tools/uuid-corrector/project.clj

```diff
@@ -0,0 +1,25 @@
+(use '[clojure.java.shell :only (sh)])
+(require '[clojure.string :as string])
+
+(defn git-ref
+  []
+  (or (System/getenv "GIT_COMMIT")
+      (string/trim (:out (sh "git" "rev-parse" "HEAD")))
+      ""))
+
+(defproject uuid-corrector "1.0.0"
```

Comment:
Template mover may not have done this (to be honest, I don't remember), but all other projects in DiscoveryEnvironmentBackend use the same version number. It's probably best to keep this convention to avoid confusing some of our dev ops scripts.